### DISCUSSION TITLE
Not working in API

### DISCUSSION THREAD

**Discussion by jainilapi** [owen-it/laravel-auditing#982] [Q&A]
When change from web it is working but not works in API

**Comment by willpower232** [comment]
Can you explain a bit more about which bit isn't working? Is it unable to identify the user perhaps?

**Reply by parallels999** [reply]
Maybe the API uses an undeclared guard 
https://github.com/owen-it/laravel-auditing/blob/23ff144bbf2bdc726ad82eb1dbd3d66504c6af49/config/audit.php#L29-L32
